sources/packages/backend/apps/db-migrations/src/migrations/1753309242467-SetResolvedRestrictionsToBeReprocessed.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class SetResolvedRestrictionsToBeReprocessed1753309242467
```

Comment:
Minor and not a blocker. Can we have the class name similar to sql file name? We never had an agreement to such thing. But mostly other migrations follow it.

Reply:
File renamed and PR description updated.